pyproject.toml

```diff
@@ -2,8 +2,6 @@
 requires = ["setuptools>=60", "wheel", "setuptools_scm>=6.2", "grpcio", "grpcio-tools>=1.47.0", "mypy-protobuf==3.1", "sphinx!=4.0.0"]
 build-backend = "setuptools.build_meta"
 
-[tool.setuptools_scm]
```

Comment:
hmm.. just read about setuptools_scm. Seems useful in general and probably it is for for a reason. I would avoid changing files without proposing similar change in upstream repo.